src/main/java/org/jabref/gui/importer/fetcher/WebSearchPaneViewModel.java

```diff
@@ -130,16 +144,27 @@ public void search() {
         }
 
         SearchBasedFetcher activeFetcher = getSelectedFetcher();
+        BackgroundTask<ParserResult> task;
```

Comment:
This variable can be introuced in line 162, can't it? In JabRef, we introduce the variable at the latest point possible - not somewhere inbetween.